### PR TITLE
Fix merge conflicts

### PR DESCRIPTION
**Pull request by phl0**

### CODE REVIEW

File: application/views/user/edit.php [hunk 1]
```diff
@@ -204,17 +204,10 @@
 										<label for="user_measurement_base"><?= __("Measurement preference"); ?></label>
 										<?php if(!isset($user_measurement_base)) { $user_measurement_base='M'; }?>
 										<select class="form-select" id="user_measurement_base" name="user_measurement_base" aria-describedby="user_measurement_base_Help" required>
-<<<<<<< alpha
 											<option value ''></option>
```
Comment (int2001) [inline application/views/user/edit.php:207]: can you fix the missing `=` here?

**Review by int2001 — CHANGES_REQUESTED** [review]
L207 was the only issue i found.
rest looks good

File: application/views/user/edit.php (hunk 1 above)
Comment (phl0) [reply application/views/user/edit.php]: Tnx. Fixed.

Comment (int2001) [reply application/views/user/edit.php]: tnx

**Review by int2001 — APPROVED** [review]

**Review by HB9HIL — APPROVED** [review]

**Pull request merged**